cmd/flags.go

```diff
@@ -56,6 +56,14 @@ var globalFlags = []cli.Flag{
 	},
 }
 
+// Flags common across all I/O commands such as cp, mirror, stat, pipe etc.
+var ioFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "encrypt-key",
+		Usage: "encrypt/decrypt objects (using server-side encryption client provided keys)",
```

Comment:
might be better to use `customer provided keys` in lieu of `client provided keys`